src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/WebClientConfiguration.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.config
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.reactive.function.client.WebClient
+
+@Configuration
+class WebClientConfiguration(
+  @Value("\${hmpps.auth.url}") private val oauthApiUrl: String,
+  private val webClientBuilder: WebClient.Builder
+) {
+
+  @Bean
+  fun oauthApiHealthWebClient(): WebClient {
+    return webClientBuilder.baseUrl(oauthApiUrl).build()
+  }
+}
```

Comment:
Configures a web client for `hmpps-auth`. This may expand to add a few more, e.g. `prison-api` as and when this service needs to call others.